readme.md

```diff
@@ -145,7 +145,9 @@ add_filter('bladerunner/cache/path', function() {
 
 If you don't want Bladerunner to create the cache folder:
 ```php
-add_filter('bladerunner/cache/make', false);
+add_filter('bladerunner/cache/make', function() {
```

Comment:
This could just be `add_filter( 'bladerunner/cache/make', '__return_false' );`

See https://developer.wordpress.org/reference/functions/__return_false/